wordpress.org/public_html/wp-content/plugins/wporg-gp-translation-suggestions/inc/class-plugin.php

```diff
@@ -187,4 +188,32 @@ public function extend_translation_suggestions( $entry ) {
 		</details>
 		<?php
 	}
+
+	/**
+	 * Update the number of external translations used.
+	 *
+	 * @param object $translation Created translation.
+	 */
+	public function update_external_translations( $translation ) {
+		if ( 'GP_Route_Translation' === GP::$current_route->class_name && 'translations_post' === GP::$current_route->last_method_called ) {
```

Comment:
Just an idea to keep the levels of indentation down:
```suggestion
		if ( 'GP_Route_Translation' !== GP::$current_route->class_name || 'translations_post' !== GP::$current_route->last_method_called ) return;
```